core/src/main/java/org/bitcoinj/script/Script.java

```diff
@@ -1297,7 +1333,7 @@ private static void executeCheckSig(Transaction txContainingThis, int index, Scr
             // This RuntimeException occurs when signing as we run partial/invalid scripts to see if they need more
             // signing work to be done inside LocalTransactionSigner.signInputs.
             if (!e1.getMessage().contains("Reached past end of ASN.1 stream"))
-                log.warn("Signature checking failed! {}", e1.toString());
+                log.warn("Signature checking failed! {}", e1);
```

Comment:
I think in this case the "{}" stays unused, because the warn(String, Throwable) overload is used.